### PR TITLE
Add cache to roles and permissions

### PR DESCRIPTION
**Pull request by rafaelpv**

### CODE REVIEW

File: src/Kodeine/Acl/Traits/HasRole.php [hunk 1]
```diff
@@ -63,7 +71,6 @@ public function is($slug, $operator = null)
         $operator = is_null($operator) ? $this->parseOperator($slug) : $operator;
 
         $roles = $this->getRoles();
-        $roles = $roles instanceof \Illuminate\Contracts\Support\Arrayable ? $roles->toArray() : (array) $roles;
```
Comment (kodeine) [inline src/Kodeine/Acl/Traits/HasRole.php]: may i know why this was removed? would it compromise laravel < 5.3?

Comment (rafaelpv) [inline src/Kodeine/Acl/Traits/HasRole.php]: Sorry, my bad. It's already fixed, thanks

Comment (kodeine) [inline src/Kodeine/Acl/Traits/HasRole.php]: Thanks a lot, i'm going to merge this.

**Comment by peetersdiet** [comment]
@kodeine I did a quick test on Laravel 5.3 and no problems here. As stated before, we only use the HasRole Trait, so the test is valid only for code affected by that trait.

**Comment by bytestream** [comment]
This causes problems when upgrading from previous versions. The cache is never cleared when updating models (ignoring the fact that by default the cache expires every minute). Also worth noting that the config item doesn't exist when upgrading (unless explicitly added by the user), and ````NULL```` in ````Cache::remember```` translates to forever.

**Pull request merged**